Octokit.Reactive/Clients/ObservableReleasesClient.cs

```diff
@@ -33,7 +33,7 @@ public IObservable<Release> GetAll(string owner, string name)
             Ensure.ArgumentNotNullOrEmptyString(owner, "owner");
             Ensure.ArgumentNotNullOrEmptyString(name, "name");
 
```

Comment:
Personally I'd lean towards "no", but I'll let @Haacked chime in here as well and see if we agree or not on this...